control-plane/subcommand/connect-init/command.go

```diff
@@ -177,7 +177,12 @@ func (c *Command) Run(args []string) int {
 			// it is not "lost" to the user at the end of the retries when the pod enters a CrashLoop.
 			if registrationRetryCount%10 == 0 {
 				c.logger.Info("Check to ensure a Kubernetes service has been created for this application." +
-					" If your pod is not starting also check the connect-inject deployment logs.")
+					" If your pod is not starting also check the connect-inject deployment logs." +
```

Comment:
@kschoche and I talked about this and we had an idea to log this connect-ignore recommendation only if `len(serviceList) > 2`.

So something like "Error: there are multiple Consul services registered for this pod when there should only be one. Check if there are multiple Kubernetes services pointing at this pod and add the label consul.hashicorp.com/connect-ignore: \"true\" to all but one of the services."

Reply:
yeah I think that's a great idea.

Reply:
Great idea!